src/striveneditor.css

```diff
@@ -90,11 +90,15 @@
 	position: relative;
 }
 
-.Striven-Editor .file-name-el {
+.Striven-Editor p.file-name-el {
 	font-weight: bold;
 	margin: 0;
 }
 
+.Striven-Editor p.file-name-el {
+	font-size: 16px;
```

Comment:
@fedebabrauskas Shouldn't this just be in the original class definition?

Reply:
Yup, I'll fix that!